apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep1.vue

```diff
@@ -22,6 +22,7 @@
                                           class="select-button"
                                           @click="handleClickNextStep(item)"
                                 >
+                                    <!--                                    // song-lang-->
```

Comment:
suggestion: I think this comment must change to 'TODO: change it to a translation code'. :)